openshift/setup-teardown/params/CUSTOM_SETTINGS/EAGLE_OS4/api/dev/api-3-nodejs-dc.params

```diff
@@ -0,0 +1,12 @@
+NAME=eagle-api
+GROUP_NAME=eagle-epic
+APPLICATION_DOMAIN=eagle-dev.apps.silver.devops.gov.bc.ca
```

Comment:
Just curious, when do we use app-name.apps.silver.gov.bc.ca and when do we use app-name.silver.gov.bc.ca? Or are are the other ones just missing an "apps"

Reply:
Prod and test where missing some with 'apps'. Updated.

Reply:
thanks!